tensorflow_addons/image/filters_test.py

```diff
@@ -109,3 +110,89 @@ def test_three_channels(self):
 
 if __name__ == "__main__":
     tf.test.main()
+
```

Comment:
Looks to be a duplicated `if __name__: test` call here. Can we remove the first one so it's only done once.